dist/MacOSBundleInfo.plist.in

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CFBundleInfoDictionaryVersion</key>
+	<string>6.0</string>
+	<key>CFBundlePackageType</key>
+	<string>APPL</string>
+
+	<key>CFBundleName</key>
+	<string>shadps4</string>
+	<key>CFBundleIdentifier</key>
+	<string>com.shadps4-emu.shadps4</string>
+	<key>CFBundleExecutable</key>
+	<string>shadps4</string>
+
+	<key>CFBundleVersion</key>
+	<string>1.0.0</string>
+	<key>CFBundleShortVersionString</key>
+	<string>${MACOSX_BUNDLE_SHORT_VERSION_STRING}</string>
+
+	<key>LSMinimumSystemVersion</key>
+	<string>${CMAKE_OSX_DEPLOYMENT_TARGET}</string>
```

Comment:
Have you verified this results in the correct value? I remember running into problems in the past where only certain variables are recognized in cmake templated plists.

Reply:
yes, this leads to the correct value

```sh
> grep 'LSMinimumSystemVersion' -A 1 build/shadps4.app/Contents/Info.plist
        <key>LSMinimumSystemVersion</key>
        <string>14</string>
```